rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -38,7 +38,7 @@ pub(crate) struct Managed {
 }
 
 // TODO: We're supposed to get this from Tauri, but I'd need to move some things around first
-const TAURI_ID: &str = "dev.firezone.client";
+pub const TAURI_ID: &str = "dev.firezone.client";
```

Comment:
(nit) This is called the bundle ID in apple or package ID in android. Might be nice to have a greppable constant name to be able to update these (we should really purchase our .com)

```suggestion
pub const BUNDLE_ID: &str = "dev.firezone.client";
```